### PR TITLE
Add unit tests, fix ESLint errors

### PR DESCRIPTION
**Pull request by brambrink**

### CODE REVIEW

**Review by fabianovangool — APPROVED** [review]
Looks great!

File: src/components/menu-button/__tests__/MenuButton.test.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+
+// import "@testing-library/jest-dom";
```
Comment (RubenWerdmuller) [inline src/components/menu-button/__tests__/MenuButton.test.tsx:3]: Remove log!

File: src/components/menu-button/__tests__/MenuButton.test.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import { StoreProvider } from "../../../providers/storeProvider";
+
+import { MenuButton } from "../MenuButton";
+
+describe("Menu button toggle", () => {
+  it("Press button, toggleMenu gets called", () => {
```
Comment (RubenWerdmuller) [inline src/components/menu-button/__tests__/MenuButton.test.tsx:10]: The grammer is a bit odd here. `It("Press button")`

File: src/components/team-member-modal/TeamMemberModal.tsx [hunk 3]
```diff
@@ -11,8 +11,8 @@ import { Button } from "../button";
 import { Input } from "../input";
 import { Modal } from "../modal";
 
-interface TeamMemberModal {
-  handleChange: ({ target }: React.ChangeEvent<HTMLInputElement>) => void;
+interface TeamMemberModalProps {
+  handleChange: ({ target }: ChangeEvent<HTMLInputElement>) => void;
   handleSubmit: (event: any) => void;
```
Comment (RubenWerdmuller) [inline src/components/team-member-modal/TeamMemberModal.tsx:16]: GitHub Action warns us that the `any` here is unexpected ;)

File: src/components/time-entries/TimeEntries.tsx [hunk 4]
```diff
@@ -47,18 +47,20 @@ export const TimeEntries = ({ initialTimeEntries, clients }: TimeEntriesProps) =
       const date = new Date(`${newTimeEntry.date}`).toDateString();
       const startTime = new Date(`${date} ${newTimeEntry.from}`);
       const endTime = new Date(`${date} ${newTimeEntry.to}`);
-      const duration = endTime.getTime() - startTime.getTime();
+      const _duration = endTime.getTime() - startTime.getTime();
```
Comment (RubenWerdmuller) [inline src/components/time-entries/TimeEntries.tsx:50]: The `duration` with underscore doesn't seem to be necessary anymore

File: src/components/time-entry-modal/TimeEntryModal.tsx [hunk 5]
```diff
@@ -16,7 +16,7 @@ interface TimeEntryModalProps {
   duration: string;
   handleChange: ({
     target,
-  }: React.ChangeEvent<HTMLInputElement> | React.ChangeEvent<HTMLSelectElement>) => void;
+  }: ChangeEvent<HTMLInputElement> | ChangeEvent<HTMLSelectElement>) => void;
   handleSubmit: (event: any) => void;
```
Comment (RubenWerdmuller) [inline src/components/time-entry-modal/TimeEntryModal.tsx:20]: Handle `any` type

**Review by RubenWerdmuller — CHANGES_REQUESTED** [review]
Very nice!

**Comment by RubenWerdmuller** [comment]
The Vercel build isn't working, is that something you can easily fix?

**Comment by brambrink** [comment]
> The Vercel build isn't working, is that something you can easily fix?

It is because of the environment variables, I don't think I am able to add these to the repo myself, is this correct?

File: src/components/time-entries/TimeEntries.tsx (hunk 4 above)
Comment (brambrink) [reply src/components/time-entries/TimeEntries.tsx:50]: I also use a state called duration for the duration per time entry which gives a ESLint error. I could also rename duration to something else, although I feel like this is also a good solution. What do you think?

**Pull request merged**